### PR TITLE
Fixed up directory structure on my local branch

### PR DESCRIPTION
**Pull request by jvanderaa**
This should be the updated version with all of the changes requested. We can delete the other PR.

### CODE REVIEW

**Comment by jeremystretch** [comment]
Looks like tests are failing on some invalid interface types (you can see the exact failures in the [test logs](https://github.com/netbox-community/devicetype-library/pull/16/checks?check_run_id=365235407) on GitHub).

File: device-types/Cisco/asr1002-hx.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/4
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/5
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/6
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/7
+    type: 1000base-x-sfp
+  - name: Te0/0/0
```
Comment (jeremystretch) [inline device-types/Cisco/asr1002-hx.yaml]: Needs to be the full form of the interface name (i.e. `TenGigabitEthernet`).

File: device-types/Cisco/c9300-24p.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+manufacturer: Cisco
+model: C9300-24P
+part_number: C9300-24P
+slug: c9300-24p
+u_height: 1
+is_full_depth: false
```
Comment (jeremystretch) [inline device-types/Cisco/c9300-24p.yaml]: Devices with front-to-back airflow should be considered full-depth.

File: device-types/Cisco/asr1002-hx.yaml (hunk 1 above)
Comment (jvanderaa) [reply device-types/Cisco/asr1002-hx.yaml]: Makes sense. I was going back and forth in my mind on these. A `show ip interface brief` would show it in the shortened fashion (not sure why). Versus the actual show interface. Will get that fixed up and checked on the other ones tonight.

**Pull request merged**